dev/com.ibm.websphere.appserver.webProfile-8.0/resources/l10n/com.ibm.websphere.appserver.webProfile-8.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2012 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature combines the Liberty features that support the Java EE 8.0 Web Profile.
```

Comment:
Is it "Java EE Web Profile 8.0" or "Java EE 8.0 Web Profile" ? I suspect Kevin would have caught this if i was wrong.

Reply:
this is a copy/pase of what we had for 7.0, with a version bump, so I think we should just keep in consistent